dhcpv4/modifiers.go

```diff
@@ -43,6 +43,16 @@ func WithGatewayIP(ip net.IP) Modifier {
 	}
 }
 
+// WithRelayAgentInfo copies the relay options from the request to the reply.
+func WithRelayAgentInfo(request *DHCPv4) Modifier {
+	return func(d *DHCPv4) {
+		// If request has Relay Agent Info copy it to the reply
+		if relayOpt := request.RelayAgentInfo(); relayOpt != nil {
+			d.UpdateOption(dhcpv4.Option{Code: dhcpv4.OptionRelayAgentInformation, Value: relayOpt})
```

Comment:
I know I added it it my previous comment but the dhcpv4 namespace is not needed.

Reply:
Fixed that and the function name in dhcpv4.go - have something funny going on with my test build environment bc it should not have built cleanly on my system.